client/src/main/scala/com/github/dapperware/slack/models/RemoteFile.scala

```diff
@@ -0,0 +1,40 @@
+package com.github.dapperware.slack.models
+
+import io.circe.Decoder
+import io.circe.generic.semiauto.deriveDecoder
+
+final case class RemoteFile(
+  id: String,
+  created: Long,
+  timestamp: Long,
+  name: String,
+  title: String,
+  mimetype: String,
+  filetype: String,
+  pretty_type: String,
+  user: String,
+  editable: Boolean,
+  size: Long,
+  mode: String,
+  is_external: Boolean,
+  external_type: String,
+  is_public: Boolean,
+  public_url_shared: Boolean,
+  display_as_bot: Boolean,
+  username: String,
+  url_private: String,
+  permalink: String,
+  comments_count: Int,
+  is_starred: Boolean,
+  shares: Map[String, String],
```

Comment:
doubtful of this. The example response the docs just has `shares: {}`

Reply:
This looks like a good reference since this is maintained by slack: https://github.com/slackapi/java-slack-sdk/blob/adbae3bd4cd3108babba16bab7a0882e71543270/slack-api-model/src/main/java/com/slack/api/model/File.java#L17

Reply:
Looks like it should be:

```
{
  public: { <channel-id>: [{}] },
  private: { <channel-id>: [{}] }
}
```

Reply:
done